content/en/agent/guide/how_remote_config_works.md

```diff
@@ -0,0 +1,149 @@
+
+The following diagram illustrates how Remote Configuration works:
+
+{{<img src="agent/guide/RC_Diagram_v2.png" alt="RC Diagram" width="90%" style="center">}}
+
+1. Configure select product features through the Datadog UI.
+2. The product feature configurations are securely stored within Datadog.
+3. Agents in your environments securely poll, receive, and automatically apply configuration updates from Datadog.
+
+**Note**: You cannot see configuration changes applied through Remote Configuration in your Agent configuration file.
+
+## Product and feature capabilities
+The following products and features are supported with Remote Config: 
+
+#### Application Security Management (ASM):  
+<div class="alert alert-info">This feature is in beta.</div>
+
+- **1-click ASM activation**: This feature allows you to enable ASM in 1-click from the Datadog UI.
+- **In-App Web Application Firewall (WAF) automated attack patterns updates**: This feature allows your service to receive the newest attack patterns as Datadog releases them, following new vulnerabilities or attack vectors being disclosed.
+- **Protect**: This feature allows you to block attackers' IPs, authenticated users, and suspicious requests that are flagged in ASM Security Signals/Traces temporarily or permanently through the Datadog UI.
+
+#### Application Performance Monitoring (APM):  
+<div class="alert alert-info">This feature is only available in private beta.</div>
+
+- **Remotely set agent sampling rate**: This features provides you the ability to remotely configure the Datadog Agent to change its trace sampling rates and set rules to scale your organization's trace ingestion according to your needs, without needing to restart your Datadog Agent. 
+
+#### Dynamic Instrumentation:
+<div class="alert alert-info">This feature is only available in private beta.</div>
+
+- **Dynamic Instrumentation**: This feature provides you the ability to send critical metrics, traces, and logs from your live applications with no code changes.
+
+#### Cloud Workload Security (CWS):
+<div class="alert alert-info">This feature is only available in private beta.</div>
+
+- **Automatic default agent rule updates**: This feature allows you to automatically receive and update the default agent rules maintained by Datadog as new agent detections and enhancements are released. 
+
+## Security Considerations
+ 
+Datadog has implemented the following safeguards which are designed to protect the confidentiality, integrity, and availability of configurations received and applied to your Agents and tracing libraries:
+
+* Agents deployed in your infrastructure request configurations from Datadog.
+* Datadog never sends configurations unless requested by Agents, and only sends configurations relevant to the requesting Agent.
+* Since the configuration requests are initiated from your Agents to Datadog over HTTPS (port 443), there is no need to open additional ports in your network firewall.
```

Comment:
```suggestion
* Because the configuration requests are initiated from your Agents to Datadog over HTTPS (port 443), there is no need to open additional ports in your network firewall.
```